src/components/ToDos.jsx

```diff
@@ -0,0 +1,143 @@
+import React, { Component } from "react";
+
+let toDoData = [
+  { task: "Learn JS", isComplete: false },
+  { task: "Learn HTML", isComplete: true },
+  { task: "Learn React", isComplete: true }
+];
+
+// let toDoData = window.localStorage.getItem("data")
+//   ? window.localStorage.getItem("data")
+//   : [{ task: "Learn React", isComplete: true }];
+
+export default class ToDos extends Component {
+  state = {
+    data: toDoData,
```

Comment:
```suggestion
    data: [...toDoData],
```
Copy needed array in the state.
Suggestion 2: move todoData to `.json` file and import it in your app